qemu/tests/qemu_guest_agent.py

```diff
@@ -243,6 +246,34 @@ def gagent_check_install(self, test, params, env):
         pass
 
     @error_context.context_aware
+    def gagent_check_install_uninstall(self, test, params, env):
```

Comment:
why not call gagent_install and gagent_uninstall directly

Reply:
Do you mean the if condition? If the pkg is installed already,it maybe occure error when reinstalling it which is not a regular operation.